readthedocs/doc_builder/environments.py

```diff
@@ -136,23 +126,13 @@ def __str__(self):
         return '\n'.join([self.get_command(), output])
 
     def run(self):
-        """
-        Set up subprocess and execute command.
-
-        :param cmd_input: input to pass to command in STDIN
-        :type cmd_input: str
-        :param combine_output: combine STDERR into STDOUT
-        """
+        """Set up subprocess and execute command."""
         log.info("Running: '%s' [%s]", self.get_command(), self.cwd)
 
         self.start_time = datetime.utcnow()
         stdout = subprocess.PIPE
-        stderr = subprocess.PIPE
+        stderr = subprocess.STDOUT
         stdin = None
```

Comment:
We can probably remove these variables as well, and use them directly in the call to `subprocess.Popen`, right?